### DISCUSSION TITLE
Test circuit file

### DISCUSSION THREAD

**Discussion by yimjinkyu1** [Xyce/Xyce#53] [Q&A] [answered]
I am very interested in your paper and xyce program. Currently I installed the xyce program. I want to test parallel scaling. I'd like to get the 100k, 1 miilion cmos transistor circuit files you tested in your paper. Is it possible to share the file?

**Comment by hkthorn** [comment, marked as answer]
Thank you for your interest in Xyce!  Unfortunately, those particular circuit files (netlists) cannot be disseminated outside of Sandia Laboratories.  So, we cannot share them.
